package.json

```diff
@@ -17,15 +17,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.147",
-    "@companieshouse/node-session-handler": "^5.0.1",
-    "@companieshouse/structured-logging-node": "^2.0.1",
+    "@companieshouse/api-sdk-node": "^2.0.103",
+    "@companieshouse/node-session-handler": "^4.1.9",
+    "@companieshouse/structured-logging-node": "^1.0.8",
     "cookie-parser": "1.4.5",
     "escape-html": "1.0.3",
     "express": "~4.18.2",
     "express-actuator": "^1.8.4",
     "express-validator": "^7.0.1",
-    "govuk-frontend": "^4.7.0",
```

Comment:
`4.7.0` should be compatible with v20.. curious to know why we're downgrading here

Reply:
same as above